games-simulation/simutrans/files/simutrans-0.122.0-Fix-Makefile.patch

```diff
@@ -0,0 +1,22 @@
+--- a/Makefile	2020-10-15 08:32:29.000000000 +0200
++++ b/Makefile	2020-10-23 15:57:17.517823272 +0200
+@@ -17,8 +17,8 @@
+ SDL_CONFIG      ?= sdl-config
+ SDL2_CONFIG     ?= pkg-config sdl2
+ #SDL2_CONFIG     ?= sdl2-config
+-FREETYPE_CONFIG ?= freetype-config
```

Comment:
Generally it's better not to readd commented out stuff in patch, since it unnecessarily increases patch size. But here I guess it's small enough not to bother updating it.

By the way, you could just pass an override to the `emake` call but since you do a patch anyway...